bundle/Resources/translations/ezplatform_fields_groups.fr_FR.xlf

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" datatype="plaintext" original="file.ext" target-language="fr">
+    <body>
```

Comment:
This is the content sent by crowdin.com, I think I'm gonna leave it as it is. It's probably because there are no (approved) translations for this file.